views/layouts/default.twig

```diff
@@ -47,9 +47,16 @@
             <ul class="nav navbar-nav">
                 <li><a href="{{ path('guidelines') }}">Guidelines</a></li>
                 <li><a href="{{ path('why') }}">Why Mentoring</a></li>
-                <li><a href="{{ path('mentors') }}">Search Mentors</a></li>
+                <!-- <li><a href="{{ path('mentors') }}">Search Mentors</a></li> -->
```

Comment:
This is currently commented out in production, so I commented it out in this PR.